src/devices/bus/hp_hil/hlekbd.cpp

```diff
@@ -3,8 +3,8 @@
 #include "hlekbd.h"
 
 #include "machine/keyboard.ipp"
+#include <logmacro.h>
```

Comment:
Please use quotes for these headers, and only use angle brackets for system/standard library headers.  Also, it's become convention to have a blank line followed by:

```
//#define VERBOSE 1
#include "logmacro.h"
```

to remind the next person who comes along what they need to do to enable logging.